src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.card__stars {
+  display: flex;
+}
+
+.card__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-size: 13px;
+  background-position: center;
+  background-repeat: no-repeat;
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  &:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
```

Comment:
styles related to stars BEM block itself should be applied to stars BEM block. styles related to stars in card (external geometry(margins, position etc.) of block) for card__stars element of card BEM block